### PR TITLE
IA-386: update code.json for sba-gov-packer

### PR DESCRIPTION
**Pull request by jsmigovsky**

### CODE REVIEW

File: code.json [hunk 1]
```diff
@@ -146,6 +146,37 @@
...
+              "URL": "http://scripts.sil.org/cms/scripts/page.php?item_id=OFL_web"
+            }
+          ],
+          "usageType": "openSource",
+          "exemptionText": null
+        },
+        "vcs": "git",
+        "laborHours": 0,
+        "tags": ["SBA"],
+        "contact": { "email": "Andrew.Davy@sba.gov" }
```
Comment (bushong1) [inline code.json]: Should this be us?

File: code.json [hunk 2]
```diff
@@ -146,6 +146,37 @@
...
+            {
+              "name": "SIL Open Font License 1.1",
+              "URL": "http://scripts.sil.org/cms/scripts/page.php?item_id=OFL_web"
+            }
+          ],
+          "usageType": "openSource",
+          "exemptionText": null
+        },
+        "vcs": "git",
+        "laborHours": 0,
```
Comment (bushong1) [inline code.json]: Is this including contractor hours or just gov't employee?

File: code.json (hunk 1 above)
Comment (rchillard-gov) [reply code.json]: No, Andy is the open source contact for the agency.  We're good. @bushong1

File: code.json (hunk 2 above)
Comment (rchillard-gov) [reply code.json]: Actually, laborHours here should be our estimated laborHours saved from using a shared packer template.  I'd defer to you or @jsmigovsky for an estimate on that.

Comment (jsmigovsky) [reply code.json]: hard to say, each image build is different
one build could take 10 minutes other could take 4 hours
I would have to say unknown at best...

Comment (rchillard-gov) [reply code.json]: @jsmigovsky , you're going to have to SWAG it.  If you think I have more ability than you do to estimate these things, then I appreciate your confidence in me, but I really don't know where to start!  Be conservative in your SWAG.

**Pull request closed without merging**